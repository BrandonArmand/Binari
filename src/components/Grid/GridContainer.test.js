import React from "react";
import { render, screen } from "@testing-library/react";
import "@testing-library/jest-dom";
import GridContainer from "./GridContainer";

describe("GridContainer", () => {
    it("renders snapshot", () => {
        const { container } = render(<GridContainer />);
        expect(container).toMatchSnapshot();
    });
    it("renders GridContainer", () => {
        render(<GridContainer>GridContainer</GridContainer>);
        expect(screen.getByText("GridContainer")).toBeInTheDocument();
    });
});
