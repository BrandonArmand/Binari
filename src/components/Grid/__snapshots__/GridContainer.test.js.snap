// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GridContainer renders snapshot 1`] = `
<div>
  <div
    class="MuiGrid-root makeStyles-grid-1  MuiGrid-container"
  />
</div>
`;
